PSReadLine/MultilineDisplayBlock.cs

```diff
@@ -0,0 +1,138 @@
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights reserved.
+--********************************************************************/
+
+using System;
+using System.Collections.ObjectModel;
+using System.Management.Automation;
+using Microsoft.PowerShell.Internal;
+using Microsoft.PowerShell;
+
+namespace Microsoft.PowerShell
+{
+    public partial class PSConsoleReadLine
+    {
+        private abstract class MultilineDisplayBlock
```

Comment:
I think the base class should only contain the helper methods extracted from `Menu` that can be shared for displaying random texts. There doesn't seem to be a need to define any abstract or virtual methods in the base class, as there is no scenario that we need to work with a base class instance.

Reply:
Refactored as discussed.